phpunit.xml

```diff
@@ -19,4 +19,9 @@
             <directory suffix=".php">src/Translatable</directory>
         </whitelist>
     </filter>
+    <php>
+        <env name="APP_ENV" value="testing"/>
+        <env name="DB_CONNECTION" value="sqlite"/>
```

Comment:
Great change to allow local testing.

Reply:
Thanks :) We could use a different `phpunit.xml` file for TravisCI in the future 👍